Content.Server/GameObjects/Components/Cargo/CargoConsoleComponent.cs

```diff
@@ -120,59 +122,78 @@ private void UserInterfaceOnOnReceiveMessage(ServerBoundUserInterfaceMessage ser
                         break;
                     }
+                case CargoConsoleApproveOrderMessage msg:
+                    {
+                        if (_requestOnly ||
+                            !orders.Database.TryGetOrder(msg.OrderNumber, out var order) ||
+                            _bankAccount == null)
+                        {
+                            break;
+                        }
 
-                    PrototypeManager.TryIndex(order.ProductId, out CargoProductPrototype product);
-                    if (product == null!)
-                        break;
-                    var capacity = _cargoOrderDataManager.GetCapacity(orders.Database.Id);
-                    if (capacity.CurrentCapacity == capacity.MaxCapacity)
+                        PrototypeManager.TryIndex(order.ProductId, out CargoProductPrototype product);
+                        if (product == null!)
+                            break;
+                        var capacity = _cargoOrderDataManager.GetCapacity(orders.Database.Id);
+                        if (capacity.CurrentCapacity == capacity.MaxCapacity)
+                            break;
+                        if (!_cargoConsoleSystem.ChangeBalance(_bankAccount.Id, (-product.PointCost) * order.Amount))
+                            break;
+                        _cargoOrderDataManager.ApproveOrder(orders.Database.Id, msg.OrderNumber);
+                        UpdateUIState();
                         break;
-                    if (!_cargoConsoleSystem.ChangeBalance(_bankAccount.Id, (-product.PointCost) * order.Amount))
-                        break;
-                    _cargoOrderDataManager.ApproveOrder(orders.Database.Id, msg.OrderNumber);
-                    UpdateUIState();
-                    break;
-                }
+                    }
                 case CargoConsoleShuttleMessage _:
-                {
-                    var approvedOrders = _cargoOrderDataManager.RemoveAndGetApprovedFrom(orders.Database);
-                    orders.Database.ClearOrderCapacity();
-                    // TODO replace with shuttle code
-
-                    // TEMPORARY loop for spawning stuff on top of console
-                    foreach (var order in approvedOrders)
                     {
-                        if (!PrototypeManager.TryIndex(order.ProductId, out CargoProductPrototype product))
-                            continue;
-                        for (var i = 0; i < order.Amount; i++)
+                        //var approvedOrders = _cargoOrderDataManager.RemoveAndGetApprovedFrom(orders.Database);
+                        //orders.Database.ClearOrderCapacity();
+
+                        // TODO replace with shuttle code
+                        // TEMPORARY loop for spawning stuff on telepad (looks for a telepad adjacent to the console)
+                        IEntity? cargoTelepad = null;
```

Comment:
BTW this is the only part of this file that was really changed, all these other additions and removals were just me running the formatter